### ISSUE TITLE
Spinner icon keeps rotating

### ISSUE THREAD

**Issue by ergonomicus** [laravel-shift/workbench-issues#49]
**Desktop:**
 - OS: Linux
 - OS Version: Linux Mint 20.2 Cinnamon
 - App Version: 0.7.0

**Describe the bug**
After running a Global Facade task, the cog icon keeps on spinning, though the task seems completed:
`{"event":"step-completed","data":{"step":"laravelglobal-facades","runtime":"3.027063"}}`

**To Reproduce**
1. Open Workbench desktop app (AppImage) and select 'Global Facades' tool
2. Click on 'Run'
3. The task seems to finish, facade imports are replaced, but the cog icon keeps spinning, which is confusing.

![facade](https://user-images.githubusercontent.com/11394513/142710760-f589cfeb-ab6f-46ff-9fc3-229492ecffe1.png)

**Comment by jasonmccreary** [comment]
Yes, this is a known bug which happens on all platforms. It happens when the I/O between the system and Docker get out of order. It's a hard one to solve, but its on our radar.

**Comment by jasonmccreary** [comment]
@ergonomicus, sorry for the delay. We've been busy on backend issues. I'm going to bring in @saurabh85mahajan to help us get caught up on these UI issues.
